docs/src/pages/index.tsx

```diff
@@ -91,7 +99,80 @@ function HomepageHeader() {
   );
 }
 
-function ProsCards() {
+function HowItWorks() {
+  return (
+    <div className="container margin-top--md">
+      <Heading as="h1" className="margin-bottom--md text--center text--primary">
+        How it works?
+      </Heading>
+      <p className={clsx('text--center', styles.sectionSubheading)}>
+        Send inputs as RTP streams or MP4 files. Configure mixing with HTTP requests. Get the mixed
```

Comment:
I'm not realy sure what, but maybe we could do sth different than putting that in a single line? Maybe bullet points?